cloudca/resource_cloudca_instance.go

```diff
@@ -266,9 +275,8 @@ func resourceCloudcaInstanceDelete(d *schema.ResourceData, meta interface{}) err
 	if _, err := ccaResources.Instances.Destroy(d.Id(), true); err != nil {
 		if ccaError, ok := err.(api.CcaErrorResponse); ok {
 			if ccaError.StatusCode == 404 {
-				fmt.Errorf("Instance %s does no longer exist", d.Get("name").(string))
 				d.SetId("")
-				return nil
+				return fmt.Errorf("Instance %s does no longer exist", d.Get("name").(string))
```

Comment:
I did this because we had a fmt.Errorf(..) but we weren't doing anything with the error. So I thought it was a mistake. Will fix.